### ISSUE TITLE
Does not autmatically load restored closed tabs

### ISSUE THREAD

**Issue by jtbergemann** [aghorler/LoadTabOnSelect#4]
I don't know if it's by design (or perhaps an interaction with another addon?): if I restore a closed tab with Ctrl-Shift-T that tab is restored but not loaded, so I get a tab with "Tab unloaded by LoadTabOnSelect." as the whole content, and I have to flip tabs to load that tab.

Firefox 56.0.1 (64 bit), Windows 10

**Comment by oloendithas** [comment]
Same for me

**Comment by Lateralus138** [comment]
Same for me when trying to reload tabs from the OneTab plugin. Same error message as in the OP.

**Comment by a2sheppy** [comment]
Yeah, you get this in the URL bar: "moz-extension://d22fa2fb-aba0-8149-97b5-e26191987d2a/html/tab.html?managedUrl=https://original.url.here". If you delete everything but the original URL, you get the content you expect back. This should be happening automatically.
